e2e/integration/innovationhubtests.js

```diff
@@ -1,6 +1,7 @@
 context( 'Test for Innovation Hub', () => {
```

Comment:
Rename this file to ideahub tests

Reply:
https://github.com/1-Platform/one-platform/pull/1355/commits/0027b659f74abff15b59f979f7ff3ad6d8b201c6 done @riginoommen